src/sim_adapter/uit_adapter.py

```diff
@@ -32,7 +33,7 @@ def __init__(self, sim: Simulator):
         eventlet.wsgi.server(eventlet.listen(('', 4567)), self.app)
 
     def connect(self, sid, environ):
-        print('connect', sid)
+        rospy.loginfo(f'Connect to socket id: {sid}')
```

Comment:
Please use C-style formatting and arguments instead. For example

Change
```python
rospy.loginfo(f'Connect to socket id: {sid}')
```

to:
```python
rospy.loginfo('Connect to socket id: %d', sid)  # or %s if sid is a string
```

Because if we use f-string or format ourselves, python still runs and replaces the format even when we do not set that logging level.